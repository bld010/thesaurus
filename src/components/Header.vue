<template>
  <header>
    <h1>Thesaurus</h1>
    
  </header>
</template>


<script>

  export default {
    name: 'Header',
    components: {
    },
    data: () => {
      return {

      }
    }
  }
</script>

<style scoped>

h1 {
  color: #039B77;
  border: 1px solid pink; 
}

header {
  border-bottom: 1px solid #039B77;
  margin: 0;
  padding: 10px;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
}
</style>