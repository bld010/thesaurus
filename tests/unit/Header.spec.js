import Header from '../../src/components/Header.vue';
import { shallowMount } from '@vue/test-utils';
import '@vue/cli-plugin-unit-jest'

describe('Header', () => {
  it('renders a header', () => {
    const wrapper = shallowMount(Header)
  
    expect(wrapper.find('h1').text()).toEqual('Thesaurus')
  })
})