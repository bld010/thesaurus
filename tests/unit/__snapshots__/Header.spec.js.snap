// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
<header>
  <h1>TheSaurus</h1> <img src="../assets/diplodocus.svg">
</header>
`;
